dashboard/app/controllers/dynamic_config_controller.rb

```diff
@@ -30,8 +30,7 @@ def gatekeeper_delete
 
     params.require(:feature)
     feature = params[:feature]
-    where = JSON.load(params[:where]) or {}
-    where = {} if where.nil?
+    where = JSON.load(params[:where]) || {}
```

Comment:
Though I'm not going to block submission of this PR on test coverage, I will try to get around to writing some simple tests for this controller.